src/array_api.py

```diff
@@ -67,3 +66,22 @@ def get_build_byname(db, buildname='hg38'):
         bid = dc.lastrowid
     return bid
 
+# get coverage for indels
+# pid is the dataset id
+def get_indel_coverage(db, pid):
+    from lib import get_call_coverage
```

Comment:
coding style thing: it's best to have the imports at the top of the file, not inside the function.
If you have it inside the function, I think it will define the functions only within this context, not globally.